hail_scripts/v02/load_clinvar_to_es.py

```diff
@@ -27,9 +27,11 @@
 p.add_argument("-H", "--host", help="Elasticsearch host or IP", required=True)
 p.add_argument("-p", "--port", help="Elasticsearch port", default=9200, type=int)
 p.add_argument("-i", "--index-name", help="Elasticsearch index name")
-p.add_argument("-t", "--index-type", help="Elasticsearch index type", default="variant")
+p.add_argument("-t", "--index-type", help="Elasticsearch index type", default="_doc")
 p.add_argument("-s", "--num-shards", help="Number of elasticsearch shards", default=1, type=int)
 p.add_argument("-b", "--es-block-size", help="Elasticsearch block size to use when exporting", default=200, type=int)
+p.add_argument("-u", "--es-username", help="Elasticsearch username")
+p.add_argument("-p", "--es-password", help="Elasticsearch password")
```

Comment:
CLI arguments to jobs submitted to Dataproc show up in the Cloud Console. Is that ok for this password?

Reply:
oh, no that shouldn't go to console. I thought this was just a helper script that people ran locally.  @mike-w-wilson do you know if and how this script is used?

Reply:
Same question for the `--es-password` argument to `hail_scripts/v02/update_reference_data_in_existing_index.py`.

Reply:
I don't think the seqr loading pipeline uses this script. I thought it was for gnomAD.

Reply:
okay I'll change this back to v6

Reply:
@mike-w-wilson do you know about the `update_reference_data_in_existing_index.py` scrit?

Reply:
I think it was part of the local installs @bw2  was working on. The `load_clinvar_to_esy.py` may also be.  I found this for the update index script https://github.com/macarthur-lab/hail-elasticsearch-pipelines/pull/187

Reply:
> I don't think the seqr loading pipeline uses this script anywhere. I thought it was for gnomAD.

It may have been originally, but the gnomAD browser no longer uses anything in this repository.

Reply:
these scripts are also not used by local installs, so I'll just change it back to using ES v6. And at some point we should really clean out this repo

Reply:
I have reverted back the changes in both of these scripts